lib/flat_search/olx_scraper.ex

```diff
@@ -1,28 +1,46 @@
-defmodule OlxScraper do
-  use Crawly.Spider
+defmodule FlatSearch.OlxScraper do
   alias FlatSearch.Flats
 
-  @impl Crawly.Spider
-  def base_url(), do: "https://www.olx.pl"
+  def run do
+    url = "https://www.olx.pl/nieruchomosci/mieszkania/wynajem/"
+    res = Crawly.fetch(url)
+    {:ok, document} = Floki.parse_document(res.body)
```

Comment:
Can you try to change this to the pipeline, since res is used only once?

Reply:
Should I use case do to pattern match in a pipeline?

Reply:
I don't really get how would I do it :/

Reply:
url |> Crawly.fetch() |> Map.get(:body)